corehq/apps/reports_core/static/reports_core/js/maps.js

```diff
@@ -33,6 +33,12 @@ hqDefine('reports_core/js/maps', function () {
             privates.layerControl.addTo(privates.map);
 
             new (hqImport("reports/js/maps_utils").ZoomToFitControl)().addTo(privates.map);
+            // Add MapBox wordmark and correct attributes to map
+            // See https://docs.mapbox.com/help/how-mapbox-works/attribution/
+            new (hqImport("reports/js/maps_utils").MapBoxWordMark)().addTo(privates.map);
+            // scale is now placed on the bottom right because it is easier to layout with the attributes than with the wordmark
+            L.control.attribution({position: 'bottomright'}).addAttribution('&copy; <a href="http://www.mapbox.com/about/maps/">MapBox</a> | &copy; <a href="http://www.openstreetmap.org/copyright">OpenStreetMap</a> | <strong><a href="https://www.mapbox.com/map-feedback/" target="_blank">Improve this map</a></strong>').addTo(privates.map);
```

Comment:
I don't think the `Improve this map` is needed so I had it removed here https://github.com/dimagi/commcare-hq/blob/f8f29068699aeadd0d8b0cf488acec5150926b0e/corehq/apps/cloudcare/static/cloudcare/js/form_entry/entrycontrols_full.js#L888-L889

Reply:
Removed the `Improve this map` part, and will handle adding the wordmark on a separate PR.